mitxgraders/baseclasses.py

```diff
@@ -495,6 +502,21 @@ def schema_answers(self, answer_tuple):
         schema = Schema((self.validate_single_answer,))
         return schema(answer_tuple)
 
+    def post_schema_validation(self, answer_tuple):
```

Comment:
Is `post_schema_answer_validation` too long a name? Some graders do post-schema validation that is not answer validation, so it is "post schema validation" but it is not covered by this function.

Reply:
Indeed, this is only post schema answer validation (only required because of the possibility of inferring answers). I thought about your name, but it was already leading to overly long lines! Maybe I'll call it `post_schema_ans_val`, which isn't quite so long, but still conveys the relevant information.